lib/Runtime/Base/FunctionBody.h

```diff
@@ -591,6 +591,9 @@ namespace Js
 #endif
     public:
         uint frameHeight;
+#if ENABLE_NATIVE_CODEGEN
+        bool nativeEntryPointProcessed;
```

Comment:
It was really only used, among other things, to decide whether to expire a function entry point info. LoopEntryPointInfo doesn't have an Expire method.